### ISSUE TITLE
Cannot resolve method attach()

### ISSUE THREAD

**Issue by kerbb** [EverythingMe/overscroll-decor#48]
Hi there,

I'm defining an `HorizontalOverScrollBounceEffectDecorator` that way:

`HorizontalOverScrollBounceEffectDecorator decor = new HorizontalOverScrollBounceEffectDecorator(new RecyclerViewOverScrollDecorAdapter(mRecyclerView), 2.5f, 5f, HorizontalOverScrollBounceEffectDecorator.DEFAULT_DECELERATE_FACTOR);`

When I try to run

`decor.attach();`

I got a compiler error:

> Cannot resolved method attach()

I can do `decor.detach();` though.

**Comment by bengovernment** [comment]
I am having the same issue- it's protected whereas detach is public. What's up? It's described as if it were public in the ReadMe.

Love the repo though, excellent quality. 👍

**Comment by ranbena** [comment]
@rotemmiz wdyt?
